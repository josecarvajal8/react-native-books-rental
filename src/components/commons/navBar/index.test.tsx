import React from 'react';
import { Text } from 'react-native-animatable';
import renderer from 'react-test-renderer';
import NavBar from '.';
const mockProps = {
    children: () => <Text>{'hello'}</Text>,
}
test('Render navBar component correctly', () => {
    const navBarCmp = renderer.create(<NavBar {...mockProps} />).toJSON();
    expect(navBarCmp).toMatchSnapshot();
});