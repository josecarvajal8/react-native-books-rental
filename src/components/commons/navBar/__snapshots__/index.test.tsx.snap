// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render navBar component correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "height": 100,
      "width": "100%",
      "zIndex": 100,
    }
  }
>
  <Image
    resizeMode="contain"
    source={
      Object {
        "testUri": "../../../src/assets/img/bc_nav_bar_3x.png",
      }
    }
    style={
      Object {
        "height": 200,
        "position": "absolute",
        "top": -45,
        "width": "100%",
      }
    }
  />
</RCTSafeAreaView>
`;
